_i18n/es/resources/user-guides/monero-wallet-cli.md

```diff
@@ -1,147 +1,114 @@
+En este ejemplo, `Balance` es tu balance total. El `unlocked balance` es la cantidad actualmente disponible para gastar. Transacciones recientemente recibidas requieren 10 confirmaciones en la blockchain antes de ser desbloqueadas. `unlocked dust` se refiere a muy pequeñas cantidades de salidas sin gastar que se pueden haber acumulado en tu cuenta.
 
-## Sending monero
+## Enviando Monero
 
-You will need the standard address you want to send to (a long string starting with '4'), and
-possibly a payment ID, if the receiving party requires one. In that latter case, that party
-may instead give you an integrated address, which is both of these packed into a single address.
+Necesitarás la dirección estándar a la que quieres enviar (una larga cadena que comienza con '4'), y posiblemente un ID de pago, si el equipo receptor requiere uno. En ese último caso, el equipo te debería dar una dirección integrada, que es la combinación de ambas en una dirección individual.
 
-### Sending to a standard address:
+### Enviando a una dirección estándar:
 
     transfer ADDRESS AMOUNT PAYMENTID
 
-Replace `ADDRESS` with the address you want to send to, `AMOUNT` with how many monero you want to send,
-and `PAYMENTID` with the payment ID you were given. Payment ID's are optional. If the receiving party doesn't need one, just
-omit it.
+Reemplaza `ADDRESS` con la dirección a la que deseas enviar, `AMOUNT` con cuánto Monero quieres enviar, y `PAYMENTID` con el ID de pago que fuiste provisto. Los ID de pago son opcionales. Si el equipo receptor no requiere uno, sólo omítelo.
 
-### Sending to an integrated address:
+### Enviando a una dirección integrada:
 
     transfer ADDRESS AMOUNT
 
-The payment ID is implicit in the integrated address in that case.
+El ID de pago está implícito en la dirección integrada en este caso.
 
-### Specify the number of outputs for a transaction:
+### Especificar el número de salidas para una transacción:
 
     transfer RINGSIZE ADDRESS AMOUNT
 
-Replace `RINGSIZE` with the number of outputs you wish to use. **If not specified, the default is 7.** It's a good idea to use the default, but you can increase the number if you want to include more outputs. The higher the number, the larger the transaction, and higher fees are needed.
+Reemplaza `RINGSIZE` con el número de salidas que deseas utilizar. **Si no se especifica, el valor por defecto es 7.** Es una buena idea utilizar el valor por defecto, pero puedes incrementar el número si quieres incluir más salidas. Entre más alto el número, más grande es la transacción, y se requerirá más comisión.
 
+## Recibiendo Monero
 
-## Receiving monero
+Si tienes tu propia dirección de Monero, sólo necesitas dar tu dirección estándar a alguien.
```

Comment:
I'm not sure if this happened because I used Windows NotePad or why, but, I'm not sure, I haven't asked @el00ruobuob if this is a problem or why some files have those line jumps in single paragraphs...

Reply:
ok

Reply:
Sorry to bother you @el00ruobuob can you please explain this? Or say, if it's a problem or it can be left that way? We're talking about the lines ^^'